### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by chejennifer**
- add section on using the tool
- update the prereqs for development

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,30 +11,32 @@ includes [APIs](https://docs.datacommons.org/api/) and visual tools to easily
...
+## Using the tool
 
-* The tools are built using Apache Maven version 3.8.0.
-
-  For MacOS: ```brew install maven```
-
-* The tools use protobuf and require that `protoc` be installed.
-
-  For MacOS: ```brew install protoc```
+- Make sure Java 11 is installed (you can install it from [here](https://www.oracle.com/java/technologies/javase-downloads.html#javasejdk))
```
Comment (pradh) [inline README.md:16]: 11+

File: README.md [hunk 2]
```diff
@@ -43,7 +45,7 @@ To build binary: ```mvn package```
 - which will produce  ```tool/target/datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar```
 
 To use the tool: ```java -jar <path-to-datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar>``` 
-- Its useful to create an alias like ```alias dc-import='java -jar <path-to-datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar>'```, so you can invoke the tool as ```dc-import```
+- It's useful to create an alias like ```alias dc-import='java -jar <path-to-datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar>'```, so you can invoke the tool as ```dc-import```
```
Comment (pradh) [inline README.md:48]: May be we can move this to "Using the tool" section?

File: README.md [hunk 3]
```diff
@@ -43,7 +45,7 @@ To build binary: ```mvn package```
 - which will produce  ```tool/target/datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar```
```
Comment (pradh) [inline README.md:45]: my bad ... could you s/1.0-SNAPSHOT/0.1-alpha.1/ ... (https://github.com/datacommonsorg/import/pull/16)

File: README.md [hunk 4]
```diff
@@ -11,30 +11,32 @@ includes [APIs](https://docs.datacommons.org/api/) and visual tools to easily
 explore and analyze data across different datasets without data cleaning or
 joining.
 
-## Dependencies
+## Using the tool
```
Comment (pradh) [inline README.md:14]: nit: import tool

**Review by pradh — APPROVED** [review]
Thank you!

File: README.md (hunk 4 above)
Comment (chejennifer) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (chejennifer) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (chejennifer) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (chejennifer) [reply README.md]: done

File: README.md [hunk 5]
```diff
@@ -26,7 +28,8 @@ java -jar <path-to-datacommons-import-tool-jar-with-dependencies.jar>
     - For MacOS: ```brew install maven```
 2. The tools use protobuf and require that `protoc` be installed.
     - For MacOS: ```brew install protoc```
-3. Make sure Java 11 is installed (you can install it from [here](https://www.oracle.com/java/technologies/javase-downloads.html#javasejdk))
+3. Make sure Java 11+ (but not Java 16) is installed
```
Comment (pradh) [inline README.md:31]: For development we can say 11 or 12.  Yes, for dev we can be more strict about versions.

File: README.md [hunk 6]
```diff
@@ -11,13 +11,15 @@ includes [APIs](https://docs.datacommons.org/api/) and visual tools to easily
 explore and analyze data across different datasets without data cleaning or
 joining.
 
-## Using the tool
+## Using the import tool
 
-- Make sure Java 11 is installed (you can install it from [here](https://www.oracle.com/java/technologies/javase-downloads.html#javasejdk))
+- Make sure Java 11+ (but not Java 16) is installed
```
Comment (pradh) [inline README.md:16]: This "not Java 16" might not be a constraint for just running the tool.  Perhaps you can confirm with your version?

**Review by pradh — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (chejennifer) [reply README.md]: Confirmed that Java 16 can run the tool & updated the README

**Pull request merged**